bitnami/drupal/README.md

```diff
@@ -48,69 +48,131 @@ The command removes all the Kubernetes components associated with the chart and
-| `podAnnotations`                   | Pod annotations                                       | `{}`                                                         |
-| `affinity`                         | Map of node/pod affinities                            | `{}`                                                         |
-| `metrics.enabled`                  | Start a side-car prometheus exporter                  | `false`                                                      |
-| `metrics.image.registry`           | Apache exporter image registry                        | `docker.io`                                                  |
-| `metrics.image.repository`         | Apache exporter image name                            | `bitnami/apache-exporter`                                    |
-| `metrics.image.tag`                | Apache exporter image tag                             | `{TAG_NAME}`                                                 |
-| `metrics.image.pullPolicy`         | Image pull policy                                     | `IfNotPresent`                                               |
-| `metrics.image.pullSecrets`        | Specify docker-registry secret names as an array      | `[]` (does not add image pull secrets to deployed pods)      |
-| `metrics.podAnnotations`           | Additional annotations for Metrics exporter pod       | `{prometheus.io/scrape: "true", prometheus.io/port: "9117"}` |
-| `metrics.resources`                | Exporter resource requests/limit                      | {}                                                           |
+The following table lists the configurable parameters of the Drupal chart and their default values per section/component:
+
+### Global parameters
+
+| Parameter                 | Description                                     | Default                                                 |
+|---------------------------|-------------------------------------------------|---------------------------------------------------------|
+| `global.imageRegistry`    | Global Docker image registry                    | `nil`                                                   |
+| `global.imagePullSecrets` | Global Docker registry secret names as an array | `[]` (does not add image pull secrets to deployed pods) |
+| `global.storageClass`     | Global storage class for dynamic provisioning   | `nil`                                                   |
+
+### Common parameters
+
+| Parameter           | Description                                                                  | Default                                                 |
+|---------------------|------------------------------------------------------------------------------|---------------------------------------------------------|
+| `image.registry`    | Drupal image registry                                                        | `docker.io`                                             |
```

Comment:
Don't forget to document image.debug here and in values.yaml

Reply:
It's documented, line 70. The same in values.yaml

Reply:
Oh, nice, thanks!